content/tutorials/Team13Tutorial.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Battery Indicator with ESP32
+date: 2024-11-21
+authors:
+  - name: Andy Tu
+  - name: Madeline Fruin
+  - name: William Luong
+---
+
+<img src="Team13Photos\BatteryCharged.jpg" alt="[Circuit Fritz](https://www.vectorstock.com/royalty-free-vector/fully-charged-battery-color-icon-vector-28877489)"> </img>
```

Comment:
this is not how you embed images in markdown. refer to the markdown spec listed in the instructions to learn how to do this.